tldw_Server_API/app/core/Chat/chat_history.py

```
@@ -3,6 +3,14 @@
 """
 Utility functions for persisting chat history, exporting conversations, and
 preparing media content for chat interactions.
+
+Note:
+- The primary chat persistence path for `/api/v1/chat/completions` now lives in
+  the Chat service module (`chat_service.build_context_and_messages` and
+  related helpers).
+- `save_chat_history_to_db_wrapper` is retained for legacy callers and
+  Character Chat utilities; new code should prefer the chat module’s unified
```

Ruff (0.14.5)

Docstring contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF002)